test/unit_tests/container/container.cpp

```diff
@@ -36,11 +36,6 @@ using owning_handle = handle_interface<owning_identifier<storage>>;
 }  // namespace
```

Comment:
reminder to test nrn-build-ci before merging

Reply:
I already launched it manually and it passed: https://github.com/neuronsimulator/nrn-build-ci/actions/runs/5186212735/jobs/9347034540
Is there some other check you want to make?

Reply:
right, didn't realize.